### PR TITLE
Run unit tests and Build in CDP

### PR DESCRIPTION
**Pull request by jmcs**
- Run unit tests in CDP;
- Build and push docker images with CDP;
- Change owner team in Zappr;

(The CDP job is failing because of docker push failing due to the team change).

### CODE REVIEW

File: delivery.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    # Docker
+    curl -fsSL https://delivery.cloud.zalando.com/utils/ensure-docker | sh
+- desc: Run unit tests
+  cmd: |
+    docker run -d -p 5432:5432 postgres
+    lein test
+- desc: Build and push docker image
+  cmd: |
+    lein do clean, uberjar
+    touch target/scm-source.json # TODO Remove this after removing it from Dockerfile
```
Comment (harti2006) [inline delivery.yaml:22]: why not remove it immediately and fix the Dockerfile?

**Comment by lotharschulz** [comment]
```
$ curl https://kio.stups.zalan.do/apps/kio --header "Authorization: Bearer $(zign token)"  | jq .

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   804    0   804    0     0    804      0 --:--:--  0:00:01 --:--:--   500
{
  "last_modified": "2017-09-01T16:18:05.082Z",
  "description": "Kio is STUP's application registry. It holds basic metadata for applications like name, owning team, where to find documentation and source code, and so on. It is the single source of truth on which applications exist.",
  "documentation_url": "https://github.com/zalando-stups/kio",
  "specification_url": "https://github.com/zalando-stups/kio/issues",
  "publicly_accessible": false,
  "service_url": "https://kio.stups.zalan.do",
  "name": "Kio",
  "criticality_level": 2,
  "created": "2015-07-29T15:03:01.534Z",
  "scm_url": "https://github.com/zalando-stups/kio.git",
  "last_modified_by": "hjacobs",
  "required_approvers": 2,
  "active": true,
  "id": "kio",
  "subtitle": "STUPS' application registry",
  "team_id": "automata",
  "created_by": "kio-migration",
  "specification_type": "githubissues"
}
```

**Comment by lotharschulz** [comment]
👍

File: delivery.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+build_meta:
```
Comment (dryewo) [inline delivery.yaml:1]: what about `version: "2017-09-20"`?

File: delivery.yaml (hunk 1 above)
Comment (jmcs) [reply delivery.yaml:22]: This is just so we can test everything before remove the GoCD pipeline.

File: delivery.yaml (hunk 2 above)
Comment (jmcs) [reply delivery.yaml:1]: This is still using the old format (since I just copied most of it from cdp's delivery.yaml 😳)

**Comment by dryewo** [comment]
The build is failing because CDP expects .zappr.yaml in master to contain `X-Zalando-Team: automata`

**Comment by harti2006** [comment]
:+1:

**Pull request merged**